src/player_controller.c

```diff
@@ -1028,7 +1028,7 @@ void func_80029B4C(Player *player, UNUSED f32 arg1, f32 arg2, UNUSED f32 arg3) {
     player->boundingBoxCorners[0].cornerPos[0] = player->pos[0] + sp8C[0];
     player->boundingBoxCorners[0].cornerPos[1] = player->pos[1] + sp8C[1];
     player->boundingBoxCorners[0].cornerPos[2] = player->pos[2] + sp8C[2];
-    process_collision(player, &player->boundingBoxCorners[0], sp80[0], sp80[1], sp80[2]);
+    process_collision_between_player_terrain(player, &player->boundingBoxCorners[0], sp80[0], sp80[1], sp80[2]);
```

Comment:
Hmm... Not quite set on this name need to think about it for a bit.

It might actually be something like tyre_collision_mesh_collision. But that's not great. So I think your choice of terrain is pretty good.

Reply:
`tyre_terrain_collision`?

Reply:
tyre ?

Reply:
Because it's used to check if the kart tyres are touching the ground.

Reply:
yeah but player also work ?

Reply:
player_terrain_collision could potentially work.